pypeit/scienceimage.py

```diff
@@ -75,26 +80,25 @@ class ScienceImage(processimages.ProcessImages):
     frametype = 'science'
 
     # TODO: Merge into a single parset, one for procing, and one for scienceimage
-    def __init__(self, spectrograph, file_list, bg_file_list = [], ir_redux=False, det=1, binning=None, par=None):
+    def __init__(self, spectrograph, file_list, bg_file_list=[], ir_redux=False,
+                 det=1, binning=None, par=None):
 
 
         # Setup the parameters sets for this object. NOTE: This uses objtype, not frametype!
         #self.par = pypeitpar.FrameGroupPar(objtype) if par is None else par
         self.par = spectrograph.default_pypeit_par()['scienceframe'] if par is None else par
 
-        # Start up by instantiating the process images class for reading in the relevant science files
-        processimages.ProcessImages.__init__(self, spectrograph, self.par['process'],
-                                             files=[], det=det)
+        # Start us up
```

Comment:
I think these instantiations of the combine image class should go into the proc_list function where they are actually used. They should then cease to exist after that code finishes. You are keeping extra copies of these images they way that this has been constructed one in the combined image class and one in the science image class.

Reply:
We can spin them up there.  
They aren't used anywhere else at this stage
so that's fine by me.